tests/Test.MSAL.NET.Unit/ConfidentialClientApplicationTests.cs

```diff
@@ -316,6 +316,11 @@ public void ConfidentialClientUsingCertificateTest()
                 anEvent[EventBase.ConstEventName].EndsWith("http_event") && anEvent[HttpEvent.ConstResponseCode] == "200"
                 && anEvent[HttpEvent.ConstHttpPath].Contains(EventBase.TenantPlaceHolder) // The tenant info is expected to be replaced by a holder
                 ));
+
+            Assert.IsNotNull(_myReceiver.EventsReceived.Find(anEvent => // Expect finding such an event
+                anEvent[EventBase.ConstEventName].EndsWith("token_cache_lookup") && anEvent[CacheEvent.ConstTokenType] == "at"));
+            Assert.IsNotNull(_myReceiver.EventsReceived.Find(anEvent => // Expect finding such an event
```

Comment:
another nit: consider having these be two separate tests with common setup -- then when a test fails you know exactly why